WordPress/Classes/ViewRelated/Site Creation/SiteInfo/SiteInformationWizardContent.swift

```diff
@@ -293,6 +324,35 @@ extension SiteInformationWizardContent: InlineEditableNameValueCellDelegate {
     }
 }
 
+extension SiteInformationWizardContent: UITextFieldDelegate {
+    func textFieldShouldReturn(_ textField: UITextField) -> Bool {
+        for (index, row) in Row.allCases.enumerated() {
+            guard let rowTextField = valueTextField(for: row) else {
+                let errorMessage = "We expect all rows to have `valueTextField`.  Please review the logic."
+                CrashLogging.logMessage(errorMessage, properties: nil, level: .error)
+                assertionFailure(errorMessage)
+                continue
+            }
+
+            guard rowTextField == textField else {
+                continue
+            }
+
+            let indexPath = IndexPath(row: index + 1, section: 0)
+
+            guard let nextTextField = valueTextField(at: indexPath) else {
+                goNext()
```

Comment:
Ok now this is a little confusing. 😕 I thought something was off here until I realized that goNext is to next page. Can we update the name here to goNextStep or nextStep or something like that?

Reply:
Yes of course, I'll DM you to agree on a better name.